src/wazuh_modules/vulnerability_scanner/tests/policyManager/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.12.4)
+
+project(policy_manager_unit_test)
+
+set(CMAKE_CXX_FLAGS_DEBUG "-g --coverage")
+
+
+file(GLOB POLICY_MANAGER_UNIT_TEST_SRC
+    "*.cpp")
+
+file(GLOB POLICY_MANAGER_SRC
+    "${CMAKE_SOURCE_DIR}/src/policyManager/policyManager.hpp")
+
+add_definitions(-DWAZUH_UNIT_TESTING)
+
+add_executable(policy_manager_unit_test
+    ${POLICY_MANAGER_UNIT_TEST_SRC})
+
+target_include_directories(policy_manager_unit_test PRIVATE ${POLICY_MANAGER_SRC})
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
These tests never will be run on Windows.

Reply:
Done =)